db/seeds.rb

```diff
@@ -7,20 +7,92 @@
 # file = File.read(file_path)
 # list = JSON.parse(file)
 
-# Parses JSON file with a list of local addresses.
+# List of local addresses.
 locations_file_path = File.join(Rails.root, 'app', 'assets', 'locations.json')
 locations_file = File.read(locations_file_path)
-locations = JSON.parse(locations_file)
-#---------------------------------------------------------------------------#
+@locations = JSON.parse(locations_file)
+
+# List of meals.
+meals_file_path = File.join(Rails.root, 'app', 'assets', 'meals.json')
+meals_file = File.read(meals_file_path)
+@meals = JSON.parse(meals_file)
+
+# List of meals descriptions.
+meal_descriptions_file_path = File.join(Rails.root, 'app', 'assets', 'meal_descriptions.json')
+meal_descriptions_file = File.read(meal_descriptions_file_path)
+@meal_descriptions = JSON.parse(meal_descriptions_file)
+
+# List of ingredients.
+ingredients_file_path = File.join(Rails.root, 'app', 'assets', 'ingredients.json')
+ingredients_file = File.read(ingredients_file_path)
+@ingredients = JSON.parse(ingredients_file)
+
+#############################################################################
+#--------------------------------METHODS------------------------------------#
+#############################################################################
+
+# Sets meals and their allergens / diets.
```

Comment:
Basically this just creates join tables for meals that are created based off of the meal name.